packages/core/src/App/Containers/Redirect/redirect.jsx

```diff
@@ -19,9 +21,9 @@ const Redirect = ({
         case 'signup': {
             const device_data = {
                 affiliate_token   : url_params.get('affiliate_token') || '',
-                date_first_contact: url_params.get('date_first_contact') || '',
+                date_first_contact: url_params.get('date_first_contact') || toMoment().format('YYYY-MM-DD'),
```

Comment:
Do we use server time for this and not local time? User's environment may have wrong date/time set.